checkov/github_actions/utils.py

```diff
@@ -53,7 +54,16 @@ def is_workflow_file(file_path: str | Path) -> bool:
     :return: True if the file mentioned is in a github action workflow directory and is a YAML file. Otherwise: False
     """
     abspath = os.path.abspath(file_path)
-    return WORKFLOW_DIRECTORY in abspath and abspath.endswith(("yml", "yaml"))
+    return get_workflow_dir() in abspath and abspath.endswith(("yml", "yaml"))
+
+
+def get_workflow_dir() -> str:
+    """
+    Detects os and uses different dir string
+    """
+    if os.name == 'nt':
```

Comment:
```suggestion
    if platform.system() == "Windows":
```
can you use the platform module as we do in other code areas to keep them insync 🙂 and don't forget the import at the top
```python
import platform
```